gallery/workbook/microsoft.azurestackhci-clusters.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
you'll also need to add an entry in the top level `CODEOWNERS` file where the other gallery files are listed to make sure your team "owns" this file so further changes only require your team's approvals